api/pkg/utils/utils.go

```diff
@@ -215,3 +215,32 @@ func GetTimeoutSec(objSize int64) int64 {
 
 	return tmoutSec
 }
+
+//return the duplicate values present in the string array
+func DuplicatesCheck(obj []string) []string {
```

Comment:
Use more descriptive and meaningful variable /function names.

Reply:
changed